README.md

```diff
@@ -19,8 +19,8 @@ Then, run [`packer init`](https://www.packer.io/docs/commands/init).
 packer {
   required_plugins {
     parallels = {
-      version = ">= 1.0.4"
-      source  = "github.com/hashicorp/parallels"
+      version = ">= 1.1.0"
+      source  = "github.com/Parallels/packer-plugin-parallels"
```

Comment:
Because of implementation limitations the the prefix `packer-plugin-` can not be part of the source URI. Packer will automatically prepend the prefix.  
```suggestion
      source  = "github.com/Parallels/parallels"
```

Reply:
fixed